cardano-api/src/Cardano/Api/TxMetadata.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE TypeApplications #-}
```

Comment:
We generally try to avoid this extension in this package, and use type annotations instead.

Reply:
done!